blocks/hero/hero.js

```diff
@@ -0,0 +1,37 @@
+import { buildBlock } from '../../scripts/lib-franklin.js';
+
+export default function decorate(block) {
+  const h1 = block.querySelector('h1');
+  const icon = block.querySelector('span');
+  const allPictures = block.querySelectorAll('.only-picture picture img');
+
+  const resolutions = ['1200px', '768px', '375px'];
+
+  icon.setAttribute('id', 'banner-icon');
+  const sectionTwo = document.createElement('div');
+  sectionTwo.append(icon);
+  sectionTwo.append(h1);
+  sectionTwo.setAttribute('class', 'logo-text-holder');
```

Comment:
Let's change this class to `.hero__content` to keep it with the BEM methodology as best as we can. Part of the tech debts that I'd like to clean up in Franklin is the way some of the classes are being defined.

Also, make sure to change this in the `hero.css` as well.

```suggestion
  sectionTwo.setAttribute('class', 'hero__content');
```